package.json

```diff
@@ -60,7 +60,12 @@
     "test:ci:incremental": "yarn test:ci --concurrency 3 --since $(git merge-base $CIRCLE_BRANCH origin/master)",
     "test:w3t": "lerna run test --concurrency 1 --scope=**/web3torrent "
   },
-  "workspaces": [
-    "packages/*"
-  ]
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ],
+    "nohoist": [
+      "**/ethers"
```

Comment:
Is this because a few other packages are not yet upgraded to ethers v5?

Reply:
Yup. That meant both versions of `ethers` were available which caused me to run into this [issue](https://github.com/ethers-io/ethers.js/issues/844).

It also helped me catch references to v4 of `ethers` that we hadn't removed from the `xstate-wallet` and `simple-hub`.